specification/cognitiveservices/data-plane/SpeakerRecognition/Verification/stable/2021-09-05/Verification.json

```
@@ -0,0 +1,1148 @@
+            "required": true,
+            "schema": {
+              "type": "object",
+              "format": "file"
+            }
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Created",
+            "schema": {
+              "$ref": "#/definitions/TiEnrollmentInfo"
+            }
+          },
+          "default": {
+            "description": "Failure",
+            "schema": {
+              "$ref": "#/definitions/SpeakerError"
+            },
+            "x-ms-error-response": true
+          }
+        }
+      }
+    },
+    "/text-independent/profiles/{profileId}:verify": {
+      "post": {
+        "description": "Verifies existing profiles against input audio.\n\nLimitations:\n* Minimum audio input length is **1 second**\n* Maximum audio input length is **120 seconds**\n* Minimum effective speech length (excluding silence and other \"non-speech\" frames) is **4 seconds**\n  This limitation can be disabled by setting \"ignoreMinLength\" to **true**.\n* Minimum audio Signal-to-noise ratio (SNR) is **2dB**",
+        "operationId": "TextIndependent_VerifyProfile",
+        "summary": "Verify Profile",
+        "x-ms-examples": {
+          "Successful Query": {
+            "$ref": "./examples/verify.json"
+          }
+        },
+        "tags": [
+          "text-independent"
+        ],
+        "consumes": [
+          "audio/wav; codecs=audio/pcm"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/apiVersionParam"
+          },
+          {
+            "$ref": "#/parameters/profileIdParam"
+          },
+          {
+            "in": "query",
+            "name": "ignoreMinLength",
+            "type": "boolean",
+            "description": "If true, a voice print will be created immediately for this profile regardless of how much speech is supplied or stored. Default is false.",
+            "default": false
+          },
+          {
+            "name": "audioData",
+            "in": "body",
+            "description": "Binary audio file. Supported formats are audio/wav; codecs=audio/pcm. Supports audio up to 5MB.",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "format": "file"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/RecognitionInfo"
+            }
+          },
+          "default": {
+            "description": "Failure",
+            "schema": {
+              "$ref": "#/definitions/SpeakerError"
+            },
+            "x-ms-error-response": true
+          }
+        }
+      }
+    },
+    "/text-independent/phrases/{locale}": {
+      "get": {
+        "description": "Retrieves list of supported passphrases for a specific locale.",
+        "operationId": "TextIndependent_ActivationPhrase",
```

OperationId for get method should contain "Get" or "List":
```suggestion
        "operationId": "TextIndependent_GetActivationPhrases",
```

---

done